shared/actions/route-state-storage.native.js

```diff
@@ -99,7 +99,12 @@ class RouteStateStorage {
 
   store = async (dispatch: Dispatch, getState: GetState): Promise<void> => {
     const state = getState()
-    if (!state.routeTree.loggedInUserNavigated) {
+    const routeTree = state.routeTree
+    if (!routeTree) {
```

Comment:
some stuff won't happen in practice but might as well be careful. we could theoretically change the type of I.Stack that is never empty but... this is simpler